mappings/net/minecraft/client/util/ScreenshotUtils.mapping

```diff
@@ -1,5 +1,22 @@
 CLASS cvv net/minecraft/client/util/ScreenshotUtils
 	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD b DATE_FORMAT Ljava/text/DateFormat;
+	METHOD a captureFramebuffer (IILctr;)Lcub;
+		ARG 0 width
+		ARG 1 height
+		ARG 2 framebuffer
 	METHOD a getScreenshotFilename (Ljava/io/File;)Ljava/io/File;
 		ARG 0 directory
+	METHOD a saveScreenshot (Ljava/io/File;IILctr;Ljava/util/function/Consumer;)V
+		ARG 0 directory
```

Comment:
```suggestion
		ARG 0 runDir
```

Reply:
Is this the game directory or the screenshots folder?

Reply:
The game directory.